web_src/less/_base.less

```diff
@@ -893,13 +893,13 @@ a.ui.card:hover,
     }
 
     &.blue {
-      color: var(--color-primary) !important;
+      color: var(--color-blue) !important;
 
       a {
         color: inherit !important;
 
         &:hover {
-          color: var(--color-primary-dark-1) !important;
+          color: #3876b3 !important;
```

Comment:
we should move this as css var ...

Reply:
Not this time. It just follows the `red` above.


![image](https://user-images.githubusercontent.com/2114189/169362616-d2687510-7918-44d0-9d8f-8120ef3d9794.png)